### PR TITLE
Adding confirmation wizard to command remote-fsck

### PR DESCRIPTION
**Pull request by MatheusClaudi**
```
$ ml-git datasets remote-fsck dataset-ex --wizard
Do you want to download the IPLD if it is not present in the local repository [y/N]? y
iplds: 100%|████████████████████████████████████| 1.00/1.00 [00:00<00:00, 30.8iplds/s]
blobs: 100%|████████████████████████████████████| 1.00/1.00 [00:00<00:00, 749blobs/s]
INFO - MLGit: remote-fsck -- fixed   : ipld[0] / blob[0]
INFO - MLGit: remote-fsck -- total   : ipld[0] / blob[0]
```

### CODE REVIEW

**Comment by Vieira-Carlos** [comment]
Branch has conflicts

**Comment by Vieira-Carlos** [comment]
Branch has conflicts

**Review by Vieira-Carlos — APPROVED** [review]

**Review by lucasncabral — APPROVED** [review]

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -585,6 +585,8 @@ Options:
   --retry INTEGER  Number of retries to download the files from the storage
                    [default: 2].
   --full           Show the list of fixed and unfixed blobs and IPLDs.
+  --wizard         Enable the wizard to prompt the user requesting information
+                   when needed.
```
Comment (andersonpablosilva) [inline docs/mlgit_commands.md:589]: Please, update the help message according to https://github.com/HPInc/ml-git/commit/9a4f4213f658bbc05a376d31024da20616cd14ec - https://github.com/HPInc/ml-git/pull/237#discussion_r912151927

Comment (lucasncabral) [reply docs/mlgit_commands.md]: The message was updated at [0031757](https://github.com/HPInc/ml-git/pull/236/commits/00317575e2041b060811271dc544096a49cdadc6).

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**